session/connio.go

```diff
@@ -50,12 +50,14 @@ func (c *Conn) DoLogin(ctx context.Context) {
 }
 
 func (c *Conn) HeartBeatLoop(ctx context.Context) {
+	timerDuration := time.Second * 4
+	timer := time.NewTimer(timerDuration)
```

Comment:
```suggestion
	timer := time.NewTimer(timerDuration)
	defer timer.Stop()
```
Do we need to call this explicitly? To me that comment you cite implies we do.